js/src/Image.js

```diff
@@ -34,20 +34,33 @@ var ImageView = widgets.DOMWidgetView.extend({
             SizeInDeg  : '.12',
             AllowImageSelection : true
         };
-        this.el.id = `imageViewer-${seq++}`;
-        this.model.on('change:GridOn change:SurveyKey', this.redraw, this);
+        this.model.on('change:GridOn change:SurveyKey change:FilePath', this.redraw, this);
+        this.model.on('change:colorbar', this.update_color, this);
         this.redraw = this.redraw.bind(this);
+        this.update_color = this.update_color.bind(this);
         setTimeout(this.redraw, 0);
+        setTimeout(this.update_color, 0);
```

Comment:
Keep in mind that the original sample code has everything in the 'render' function.  It also implemented redraw inefficiently.  Eventually, we should go back move the logic into its proper places and implement redraw similar to update_color.